# Generated by Django 4.0.4 on 2022-05-10 07:24

import datetime
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('manager', '0015_alter_external_transfer_transfer_date_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='appointment',
            name='time_appointment_create',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 10, 24, 11, 375739)),
        ),
        migrations.AlterField(
            model_name='external_transfer',
            name='transfer_date',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 10, 24, 11, 364739), null=True),
        ),
        migrations.AlterField(
            model_name='internal_transfer',
            name='transfer_date',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 10, 24, 11, 365737), null=True),
        ),
        migrations.AlterField(
            model_name='profile_specialist',
            name='date_create',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 10, 24, 11, 367738), null=True),
        ),
        migrations.CreateModel(
            name='Refund_to_internal_account',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('description', models.TextField()),
                ('appointment', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.appointment')),
                ('reason', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.reason_for_refund')),
                ('status', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.refusal_or_refund_status')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.customuser')),
            ],
        ),
    ]
